src/jdk.jartool/share/classes/sun/tools/jar/GNUStyleOptions.java

```diff
@@ -228,6 +228,20 @@ void process(Main jartool, String opt, String arg) {
                 }
                 boolean isHidden() { return true; }
             },
+            new Option(true, OptionType.OTHER, "--release") {
```

Comment:
I think we should create a new `OptionType` for this one. If this is put under `OTHER`, then it will end up like this in the help text:

```
Other options:

  -?, -h, --help[:compat]    Give this, or optionally the compatibility, help
      --help-extra           Give help on extra options
      --version              Print program version
      --release
```
That wouldn't be the right place for this option. Instead, I think we should create a `DESCRIBE_MODULE_OR_VALIDATE` `OptionType` and configure the necessary help text for that group and include `--release` option under that. That way it will show up something like:

```
Operation modifiers valid only in describe module and validate mode:
	--release=VERSION	...

```